src/sentry/issues/escalating.py

```diff
@@ -307,16 +309,18 @@ def _issue_category_entity(category: Optional[GroupCategory]) -> EntityKey:
 def manage_issue_states(
     group: Group,
     group_inbox_reason: GroupInboxReason,
-    event: Optional[GroupEvent] = None,
-    snooze_details: Optional[Mapping[str, Any]] = None,
-    activity_data: Optional[Mapping[str, Any]] = None,
+    event: GroupEvent | None = None,
+    snooze_details: Mapping[str, Any] | None = None,
+    activity_data: Mapping[str, Any] | None = None,
 ) -> None:
     """
     Handles the downstream changes to the status/substatus of GroupInbox and Group for each GroupInboxReason
 
     `activity_data`: Additional activity data, such as escalating forecast
     """
-    data = {"event_id": event.event_id} if event else None
+    data: Dict[str, str | Mapping[str, Any]] | None = (
```

Comment:
When you use `from future import __annotations__` you can just use `dict`.

I'm surprised to see `Dict[str, str | Mapping[str, Any]]`, I would have expected `Dict[str, str]`.

Can `event.event_id` be a dictionary?

Reply:
The `Mapping[str, Any]` comes from `snooze_details` on L359.